src/registrar/assets/sass/_theme/styles.scss

```diff
@@ -23,6 +23,8 @@
 @forward "identifier";
 @forward "header";
 @forward "register-form";
+@forward "containers";
+@forward "navigation";
```

Comment:
navigation is actually called header. You don't need a new scss file. (header is the naming convention from USWDS)

Reply:
Ah...thank you, I will combine